src/main/resources/application-local1.properties

```diff
@@ -51,11 +51,11 @@ mosip.digitalcard.crypto.partner.id=mpartner-default-digitalcard
 
 # websub authentication
 mosip.iam.adapter.clientid=mpartner-default-digitalcard
-mosip.iam.adapter.clientsecret=TLwcz72m5BwmIxlj
+mosip.iam.adapter.clientsecret=Gekvkcy690ZHVkDo
```

Comment:
Don't  hardcode client secret and commit it. It is exposing the secret to outside world.